src/coreclr/jit/jitconfigvalues.h

```diff
@@ -458,8 +458,10 @@ CONFIG_INTEGER(JitEnableRemoveEmptyTry, W("JitEnableRemoveEmptyTry"), 1)
 // Overall master enable for Guarded Devirtualization.
 CONFIG_INTEGER(JitEnableGuardedDevirtualization, W("JitEnableGuardedDevirtualization"), 1)
 
-#if defined(DEBUG)
 // Various policies for GuardedDevirtualization
+CONFIG_INTEGER(JitGuardedDevirtualizationChainLikelihood, W("JitGuardedDevirtualizationChainLikelihood"), 0x4B) // 75
+CONFIG_INTEGER(JitGuardedDevirtualizationChainStatements, W("JitGuardedDevirtualizationChainStatements"), 2)
+#if defined(DEBUG)
```

Comment:
Does it mean "how many statements are allowed between two GDV candidates"?

Reply:
Yes -- it's a (crude) limit on how much other code we're willing to duplicate.

Reply:
I'm asking because I had:
```csharp
obj.GDV();
Console.WriteLine();
obj.GDV();
```
worked

```csharp
obj.GDV();
Console.WriteLine();
Console.WriteLine();
obj.GDV();
```
did not 🙂

Reply:
I set the statement limit to 2 based on some scouting in SPMI. But given the overall size decrease it might be ok to pick a larger value.

Rough numbers of opportunities / cost vs stmt limit:

| limit | chained segments | dup count |
| -- | -- | -- |
| 0 | 29 | 0 |
| 1 | 218 | 189 |
| 2 | 262 | 251 |
| 3 | 307 | 366 |
| 4 | 313 | 390 |
| 6 | 320 | 432 |
| 8 | 321 | 440 |